docs/ui/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/react';
+import { breakpoints } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { LayoutScroll } from './LayoutScroll';
+
+type LayoutProps = PropsWithChildren<{
+  /** The content within the top bar that spans the columns */
+  header: ReactNode;
+  /** The content within the left column */
+  navigation?: ReactNode;
+  /** The content within the right column */
+  sidebar?: ReactNode;
+}>;
+
+export function Layout({ header, navigation, sidebar, children }: LayoutProps) {
+  return (
+    <div css={layoutStyle}>
+      <div css={headerStyle}>{header}</div>
+      <div css={navigationStyle}>{navigation}</div>
+      <div css={contentStyle}>
+        <LayoutScroll>
+          <div css={innerContentStyle}>{children}</div>
+        </LayoutScroll>
+      </div>
+      <div css={sidebarStyle}>{sidebar}</div>
+    </div>
+  );
+}
+
+const layoutStyle = css({
+  display: 'grid',
+  height: '100vh',
+  gridTemplateRows: '60px calc(100vh - 60px)',
+  gridTemplateColumns: 'min-content auto min-content',
```

Comment:
```suggestion
  gridTemplateColumns: '280px auto 280px',
```

We may want to make this even more forceful to ensure that child pages cannot change the size of `min-content`.

Reply:
I had problems with the responsiveness of the grid with these values. When we don't render the right sidebar at all, we probably still want to use that space. That doesn't work with these fixed values, and `fit-content(280px)` does not actually limits the column's width.

Reply:
For now, I'll merge this without the suggestion, we can circle back to it later. I want to start composing the new designs with sidebars to prepare for tweaking and polishing.